include/RED4ext/Scripting/Natives/AnimKeyFrames.hpp

```diff
@@ -0,0 +1,67 @@
+#pragma once
+
+#include <cstdint>
+
+namespace RED4ext
+{
+namespace anim
+{
+enum KeyFrameComponent
+{
+    Translation = 0,
+    Rotation,
+    Scale,
+};
+
+struct KeyFrameHeader
+{
+    uint16_t timeNormalized;     // 00
+    uint16_t boneIndex : 13;     // 02
+    uint16_t componentType : 2;  // 0E
+    uint16_t rotationWSign : 1;  // 0F
+};
+RED4EXT_ASSERT_SIZE(KeyFrameHeader, 0x4);
```

Comment:
I would add this since it is a bitfield:
```suggestion
RED4EXT_ASSERT_SIZE(KeyFrameHeader, 0x4);
RED4EXT_ASSERT_OFFSET(KeyFrameHeader, timeNormalized, 0x0);
RED4EXT_ASSERT_OFFSET(KeyFrameHeader, boneIndex, 0x2);
RED4EXT_ASSERT_OFFSET(KeyFrameHeader, componentType, 0xE);
RED4EXT_ASSERT_OFFSET(KeyFrameHeader, rotationWSign, 0xF);
```